backends/tc/ebpfCodeGen.cpp

```diff
@@ -1980,6 +1990,12 @@ void ControlBodyTranslatorPNA::processMethod(const P4::ExternMethod *method) {
         auto pna_ctr = dynamic_cast<EBPFCounterPNA *>(counterMap);
         pna_ctr->emitMethodInvocation(builder, method, this);
         return;
+    } else if (declType->name.name == "Meter") {
+        auto meter = control->to<EBPF::EBPFControlPSA>()->getMeter(name);
+        auto pna_meter = dynamic_cast<EBPFMeterPNA *>(meter);
+        ::P4::warning(ErrorType::WARN_UNUSED, "This Meter (%1%) return value is not used!", name);
```

Comment:
unused warning is issued without checking, is it expected here?

Reply:
yes, this warning is emitted if meter execute() is called and the return value is not used (not assigned to some variable)